<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BotEditorForm</class>
 <widget class="QWidget" name="BotEditorForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1121</width>
    <height>727</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактор бота</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="PropertiesView" name="bot_params_view"/>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="add_message_button">
           <property name="text">
            <string>Add message</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="delete_message_button">
           <property name="text">
            <string>Delete message</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="mark_as_start_button">
           <property name="text">
            <string>Mark as start</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="generate_bot_button">
           <property name="text">
            <string>Generate bot</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="start_bot_button">
           <property name="text">
            <string>Start bot</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="stop_bot_button">
           <property name="text">
            <string>Stop bot</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGraphicsView" name="graphics_view"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="apply_button">
       <property name="text">
        <string>Apply changes</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PropertiesView</class>
   <extends>QTreeView</extends>
   <header>desktop_constructor_app.common.properties_view</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
